static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -0,0 +1,198 @@
+  ));
+
+  cards?.push(
+    <MemberCard key="see-more" data-test-id="see-more-card">
+      <MemberCardContent>
+        <MemberCardContentRow>
+          <SeeMore>
+            {tct('See all [missingMembersCount] missing members', {
+              missingMembersCount: missingMembers?.users.length,
+            })}
+          </SeeMore>
+        </MemberCardContentRow>
+        <Subtitle>
+          {tct('Accounting for [totalCommits] recent commits', {
+            totalCommits: missingMembers?.users.reduce(
+              (acc, curr) => acc + curr.commitCount,
+              0
+            ),
+          })}
+        </Subtitle>
+      </MemberCardContent>
+      <Button
+        size="sm"
+        priority="primary"
+        // TODO(cathy): open up invite modal
+        data-test-id="view-all-missing-members"
+      >
+        {t('View All')}
+      </Button>
+    </MemberCard>
+  );
+
+  return (
+    <StyledCard data-test-id="invite-banner">
+      <CardTitleContainer>
+        <CardTitleContent>
+          <CardTitle>{t('Bring your full GitHub team on board in Sentry')}</CardTitle>
+          <Subtitle>
+            {tct('[missingMemberCount] missing members that are active in your GitHub', {
+              missingMemberCount: missingMembers?.users.length,
+            })}
+            <Tooltip title="Based on the last 30 days of commit data">
+              <IconInfo size="xs" />
+            </Tooltip>
+          </Subtitle>
+        </CardTitleContent>
+        <ButtonContainer>
+          <Button
+            priority="primary"
+            size="xs"
+            // TODO(cathy): open up invite modal
+            data-test-id="view-all-missing-members"
+          >
+            {t('View All')}
+          </Button>
+        </ButtonContainer>
+      </CardTitleContainer>
+      <MemberCardsContainer>{cards}</MemberCardsContainer>
+    </StyledCard>
+  );
+}
+
+export default withOrganization(InviteBanner);
+
+const StyledCard = styled(Card)`
+  padding: ${space(2)};
+  display: flex;
+  overflow: hidden;
+`;
+
+const CardTitleContainer = styled('div')`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const CardTitleContent = styled('div')`
+  display: flex;
+  flex-direction: column;
+`;
+
+const CardTitle = styled('div')`
+  font-size: ${p => p.theme.fontSizeLarge};
+  font-weight: bold;
+  color: ${p => p.theme.gray400};
+`;
+
+const Subtitle = styled('div')`
+  font-size: ${p => p.theme.fontSizeSmall};
+  font-weight: 400;
+  color: ${p => p.theme.gray300};
+  display: flex;
+  align-items: center;
+  & > *:first-child {
+    margin-left: ${space(0.5)};
+    display: flex;
+    align-items: center;
+  }
+`;
+
+const ButtonContainer = styled('div')`
+  display: grid;
+  grid-auto-flow: column;
+  grid-column-gap: ${space(1)};
+`;
+
+const MemberCard = styled(Card)`
+  padding: ${space(2)} 18px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin: ${space(1)} ${space(0.5)} 0 0;
+  min-width: 330px;
```

Comment:
The scrolling UX is a tricky pattern. Setting an absolute width means at some widths, the scrolling is not apparent to the user.

```js
const MemberCardsContainer = styled('div')`
  position: relative;
  display: flex;
  overflow-x: scroll;
`;

const MemberCard = styled(Card)`
  width: 20%
  ...
`;
```

This will always ensure that 1 card is partially show, which indicates to the user that it can be scrolled.

We should consider making this responsive.

Reply:
i think responsiveness is still relevant for the carousel, because the carousel will scroll for the components that are nested in it